# This file was modifed by modify_ac_library.py
actions:
- ext_angular:
  - 0.0
  - 0.0
  - 0.0
  ext_duration: 1.0
  ext_force: 50.0
  ext_linear:
  - 0.0
  - 0.0
  - 0.08
  ext_torque: 4.0
  grasp_angular:
  - 0.0
  - 0.0
  - 0.0
  grasp_duration: 0.0
  grasp_force: 15.0
  grasp_linear:
  - 0.0
  - 0.0
  - 0.0
  grasp_torque: 4.0
  pre_force: 20.0
  pre_offset:
  - 0.0
  - 0.0
  - -0.02
  pre_pos:
  - 0.0
  - 0.0
  - 1.0
  pre_quat:
  - 0.0
  - 0.7071068
  - -0.7071068
  - 0.0
  pre_torque: 4.0
- ext_angular:
  - 0.0
  - 0.0
  - 0.0
  ext_duration: 1.0
  ext_force: 50.0
  ext_linear:
  - 0.0
  - 0.0
  - 0.08
  ext_torque: 4.0
  grasp_angular:
  - 0.0
  - 0.0
  - 0.0
  grasp_duration: 0.0
  grasp_force: 15.0
  grasp_linear:
  - 0.0
  - 0.0
  - 0.0
  grasp_torque: 4.0
  pre_force: 20.0
  pre_offset:
  - 0.0
  - 0.0
  - -0.02
  pre_pos:
  - 0.0
  - 0.0
  - 1.0
  pre_quat:
  - -0.174941
  - 0.6851245
  - 0.6851245
  - -0.174941
  pre_torque: 4.0
- ext_angular:
  - 0.0
  - 0.0
  - 0.0
  ext_duration: 1.0
  ext_force: 50.0
  ext_linear:
  - 0.0
  - 0.0
  - 0.08
  ext_torque: 4.0
  grasp_angular:
  - 0.0
  - 0.0
  - 0.0
  grasp_duration: 0.0
  grasp_force: 15.0
  grasp_linear:
  - 0.0
  - 0.0
  - 0.0
  grasp_torque: 4.0
  pre_force: 15.0
  pre_offset:
  - -0.01
  - 0.0
  - -0.02
  pre_pos:
  - -0.4949747468305832
  - 0.0
  - 0.6363961030678928
  pre_quat:
  - 0.1379497
  - 0.6935199
  - 0.6935199
  - 0.1379497
  pre_torque: 4.0
- ext_angular:
  - -0.2337091737817557
  - -0.16295875231130133
  - 0.09393863494533573
  ext_duration: 1.7751092186081665
  ext_force: 50.0
  ext_linear:
  - 0.0026763823861303358
  - -0.0039894406198535655
  - 0.02801145451759559
  ext_torque: 4.0
  grasp_angular:
  - 0.021842282547490127
  - 0.2921079172299172
  - -0.06477560794554887
  grasp_duration: 1.0006132793330564
  grasp_force: 30.922998802908683
  grasp_linear:
  - 0.01348817941773009
  - 0.007604004210222313
  - -0.03351692732677677
  grasp_torque: 1.0
  pre_force: 2.0181984644444464
  pre_offset:
  - 0.006061929708926245
  - 0.007953175982212223
  - -0.02
  pre_pos:
  - 0.04736757613920517
  - 0.06214566762976613
  - 0.09712448722679744
  pre_quat:
  - 0.1992894
  - 0.9235141
  - -0.1772749
  - -0.275643
  pre_torque: 4.0
- ext_angular:
  - -0.29940023070845145
  - 0.018936788724187546
  - -0.0009486224441275935
  ext_duration: 2.224894712388526
  ext_force: 50.0
  ext_linear:
  - -0.007221750355917799
  - 0.000984312510690585
  - 0.026957998153735216
  ext_torque: 4.0
  grasp_angular:
  - 0.043164810768924776
  - 0.29161461783103754
  - 0.05565710896678695
  grasp_duration: 0.9542838029084819
  grasp_force: 20.628485289480114
  grasp_linear:
  - 0.019235823439093823
  - -0.0102442601628834
  - -0.030280814265481275
  grasp_torque: 1.0
  pre_force: 1.9680640809243017
  pre_offset:
  - 0.002418366523682367
  - 0.009703169758235321
  - -0.02
  pre_pos:
  - 0.01215432550886536
  - 0.04876658792389915
  - 0.06980903419187809
  pre_quat:
  - 0.1193524
  - 0.9860264
  - -0.0486593
  - -0.1055426
  pre_torque: 4.0
- ext_angular:
  - -0.28955379864722297
  - 0.051610208477091614
  - -0.05911839028521166
  ext_duration: 0.7226081163180563
  ext_force: 50.0
  ext_linear:
  - 0.004180955980068812
  - 0.0018657560430278416
  - 0.07482959078248409
  ext_torque: 4.0
  grasp_angular:
  - -0.13851504223214214
  - 0.12995252289240805
  - 0.23221956177145373
  grasp_duration: 1.6446366337005425
  grasp_force: 4.711684426093214
  grasp_linear:
  - 0.02324204444030554
  - -0.0016801877099561922
  - -0.009129031531062712
  grasp_torque: 1.0
  pre_force: 1.8661167056580217
  pre_offset:
  - -0.008915818024062005
  - 0.004528596798326286
  - -0.02
  pre_pos:
  - -0.09330950413838413
  - 0.04739454310912268
  - 0.07465685505482798
  pre_quat:
  - 0.3649811
  - 0.7727205
  - 0.4891018
  - 0.1745601
  pre_torque: 4.0
- ext_angular:
  - -0.18223860895737007
  - 0.23784986442981917
  - 0.014715005810375568
  ext_duration: 1.4295090302019482
  ext_force: 50.0
  ext_linear:
  - -0.005076070614061674
  - 0.006029007710886375
  - 0.04106996731042198
  ext_torque: 4.0
  grasp_angular:
  - 0.22201924830514225
  - 0.05517777943959449
  - 0.19406923001376358
  grasp_duration: 0.4687652832583936
  grasp_force: 9.46219200015453
  grasp_linear:
  - 0.029761834260402215
  - 0.0034030675293683183
  - -0.05213803299703455
  grasp_torque: 1.0
  pre_force: 1.4532427864459323
  pre_offset:
  - -0.0045319848940434705
  - 0.008914096304178108
  - -0.02
  pre_pos:
  - -0.042610413948224124
  - 0.08381169451702956
  - 0.06859996598659442
  pre_quat:
  - 0.0549787
  - 0.9943871
  - -0.0646521
  - 0.0631807
  pre_torque: 4.0
